### DISCUSSION TITLE
Nft, Staking, defi systems and features should be added

### DISCUSSION THREAD

**Discussion by Hakanbitlis** [IBAX-io/go-ibax#979] [Ideas] [answered]
### Clear and concise description of the problem

It should have an easy mobile interface, nft, etf, defi mining feature should be introduced.  Also, transfer feature from bank to wallet should be added.

### Suggested solution

It should have an easy mobile interface, nft, etf, defi mining feature should be introduced.  Also, transfer feature from bank to wallet should be added.

### Alternative

It should have an easy mobile interface, nft, etf, defi mining feature should be introduced.  Also, transfer feature from bank to wallet should be added.

### Additional context

It should have an easy mobile interface, nft, etf, defi mining feature should be introduced.  Also, transfer feature from bank to wallet should be added.

### Validations

- [X] Read the [docs](https://docs.ibax.io).
- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.

**Comment by ibaxio** [comment, marked as answer]
The innovation of the IBAX blockchain network is creation of the metaverse world. It is committed to mapping the real society to the metaverse world beyond the real-world nodes and space constraints. To maintain the long-term development of the metaverse, IBAX needs a network where a huge number of creators build a metaverse together. Therefore, 18.75% of all coins is reserved to reward creators of outstanding NFTs for the metaverse. An address of each creator may only own up to 10 NFTs. Each NFT has the Power of 1 to 100. The higher this value, the higher the possibility to get the metaverse rewards.

**There are three ways to get NFTs:**

1. New accounts can get a one-time metaverse bonus. Users who deposit coins to the account and lock them for staking for 1 month instantly get one freshly minted metaverse NFT.
2. Users can fuse two NFTs in their account into one new NFT. The resulting Power of the new NFT may be either greater or smaller than the sum of Power values of the two original NFTs.
3. Users can go to a decentralized NFT trading platform to buy NFTs with a high Power value fused by other users and transfer them to their account. The launch of IBAX mainnet will enable the reward mechanism for NFT creators. Its aim is to promote the metaverse development and reach the mature metaverse world as quickly as possible.

**Rules:**

- Users can create an account in IBAX Network free of charge. Each account gets one NFT of 1.00 Power.
- Users who stake 10,000 IBXC for 30 days can join IBXC mining using the NFT of 1.00 Power within these 30 days.
- After 30 days of staking, the initial NFT of 1.00 Power can be transferred/exchanged (i.e., it becomes a free NFT).
- Two free NFTs can be fused on IBAX Network according to the fusing rules to create a new free NFT with a different Power value.
- One account can own up to 10 NFTs.

DEFI, GAMEFI and other features would be new ECOsystems of IBAX Network.
